### ISSUE TITLE
T2T-ViT-14 Backbone

### ISSUE THREAD

**Issue by MertCokelek** [MertCokelek/VisualSaliencyTransformer#1]
nn.Unfold for soft split
nn.Linear

**performer**
    nn.dropout
    nn.LayerNorm
    nn.GeLU
    nn.init.orthogonal
    torch.einsum

**Comment by MertCokelek** [comment]
In pretrained model, **transformer** is used, instead of **performer**. Both are implemented, transformer is assigned as default.

**Comment by MertCokelek** [comment]
Einsum may not be differentiable. According to the training process, **bmm** will be used instead.

**Comment by MertCokelek** [comment]
unfold and fold operations do not require autograd, so for now, pycall will be used. Will be updated later.
